README.md

```diff
@@ -1 +1,82 @@
+
+        * Release notes will be provided for any changes concerning released content, and any revisions will be backward compatible with the prior version to the degree possible
+
+      * `iof-av:Provisional`
+
+        * Indicates an ontology or construct that is considered to be under development
+
+        * Provisional content is subject to change and may change substantially before release. IOF users should be aware that it is not dependable but could be used for reference and as the basis for further work
+
+# Current Ontology Status
+
+## Released and Provisional Ontologies
+
+* `core/Core` **[Released]**
+  >  The IOF Core Ontology contains notions found to be common across multiple manufacturing domains. This file is an RDF rendering of these notions expressed in OWL. The ontology utilizes the Basic Formal Ontology or BFO as a top-level ontology but also borrows terms from various domain-independent or mid-level ontologies. The purpose of the ontology is to serve as a foundation for ensuring consistency and interoperability across various domain-specific reference ontologies the IOF publishes.
+  * `core/meta/AnnotationVocabulary` **[Released]**
+    > The IOF Annotation Vocabularyprovides a set of OWL annotation properties for annotating IOF or other content with metadata to facilitate user and ontology developer understanding.	
+  * `commons-to-core-mapping/MappingCommonsToIOF` **[Released]**
+    > This ontology maps the OMG Common Ontologies concepts to those of IOF Core.
+  * `commons-to-core-mapping/meta/MappingAnnotationVocabularyToCommons.rdf` **[Released]**
+    > The IOF Mapping Annotation Vocabulary to Commons maps the AV to the Object Management Group (OMG)&apos;s Annotation Vocabulary provided in the Commons Ontology Library. The Commons Annotation Vocabulary is a subset of what is included in the IOF AV, but is used across a number of OMG and other emerging standards and thus the mapping makes it easier to use other Commons library ontologies in an IOF context.
+* `maintenance/Maintenance` **[Provisional]**
+  > The purpose of this IOF Maintenance Reference ontology is to support semantic interoperability through the use of modular ontologies in the maintenance domain. This Ontology contains terms identified as common in a number of application ontologies for maintenance management, maintenance procedures, asset failure, and failure modes and effects analysis. The ontology is based on the IOF Core Ontology.
+* `supplychain/SupplyChainReferenceOntology` **[Provisional]**
+  > Supply Chain Reference Ontology (SCRO) aims to extend the IOF Core with the constructs (classes and properties) related to the domain of supply chain and logistics. The purpose of the ontology is to serve as a foundation for ensuring consistency and interoperability across various supply chain and logistics application ontologies.
+## Reasoner
```

Comment:
This section should ideally cover the testing methods we apply to released ontologies, which would include running the reasoner. Maybe copy in what I had written in the Readme for Core and we delete that section from Core. But this is not a show stopper as far as I'm concerned so I still approve this whole document it currently is.